server/src/api/mapController.ts

```diff
@@ -6,13 +6,29 @@ const router: express.Router = express.Router();
 
 router.get('/polygon', (async (req: Request, res: Response) => {
   const { scale, big, medium, small } = req.query;
-  const polygon = await mapService.queryPolygon(
-    Number(scale),
-    big as string,
-    medium as string,
-    small as string,
-  );
-  res.json(polygon);
+  if (scale && (big || medium || small)) {
+    const polygon = await mapService.queryPolygon(
+      Number(scale),
+      big as string,
+      medium as string,
+      small as string,
+    );
+    res.json(polygon);
+  } else {
+    res.json([
+      { name: '', path: [], code: '', codeLength: 0, center: [], type: '' },
```

Comment:
개인적인 궁금증으로 return이 아닌 해당 값을 res.json으로 보내주는지 궁금합니다.
```suggestion
      { name: '', path: [], code: '', codeLength: 0, center: [], type: '' },
```